bin/kafka-broker-versions.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+exec $(dirname $0)/kafka-run-class.sh kafka.admin.BrokerVersionCommand "$@"
```

Comment:
I was thinking that eventually we might want to have a way to request API versions from only specific brokers.  We may also want to have a way to pull the version string out of the requests they return.   I agree that it's probably safe to drop the requested list-api-versions option.  I would rather not add "api" to the name, since there may eventually be more to versioning, such as reading the version string sent over the wire, etc.